lib/log.js

```diff
@@ -0,0 +1,42 @@
+'use babel'
+
+import commandLog from './command-log'
+import {learnCo} from './config'
+import {post, patch} from './http'
+import token from './token'
+import version from './version'
+
+export default {
+  defaultData() {
```

Comment:
prob don't need this as part of the public interface (could be private fn inside hidden in this module